plugins/woocommerce/templates/single-product/product-attributes.php

```diff
@@ -23,10 +23,10 @@
 	return;
 }
 ?>
-<table class="woocommerce-product-attributes shop_attributes">
+<table class="woocommerce-product-attributes shop_attributes" aria-label="<?php echo esc_html_e('Product Details', 'woocommerece'); ?>">
```

Comment:
The `esc_html_e` function directly echoes things, so no need to have an `echo` statement before that. In addition, wondering if we want to use the `esc_attr` functions here instead since this is an attribute:

```suggestion
<table class="woocommerce-product-attributes shop_attributes" aria-label="<?php esc_attr_e( 'Product Details', 'woocommerce' ); ?>">
```

Reply:
Thanks @dkotter good point .. commit accepted ;)